gc/PKGBUILD

```diff
@@ -3,18 +3,18 @@
 
 pkgbase=gc
 pkgname=('libgc' 'libgc-devel')
-pkgver=8.2.4
-pkgrel=2
+pkgver=8.2.6
+pkgrel=1
 pkgdesc="A garbage collector for C and C++"
 arch=('i686' 'x86_64')
 groups=('libraries')
 url="https://www.hboehm.info/gc/"
 msys2_repository_url="https://github.com/ivmai/bdwgc"
 license=('MIT')
 makedepends=('autotools' 'gcc')
-source=("https://github.com/ivmai/bdwgc/releases/download/v${pkgver}/${pkgbase}-${pkgver}.tar.gz")
+source=("${msys2_repository_url}/releases/download/v${pkgver}/${pkgbase}-${pkgver}.tar.gz")
```

Comment:
minor nitpick, such changes make the PKGBUILD just harder to read

Reply:
Okay, I will change it back on the release next update.

Reply:
Done in https://github.com/msys2/MSYS2-packages/pull/4895